dfc/callstats.go

```diff
@@ -73,13 +74,16 @@ func (c *CallStatsServer) Start() {
 
 // Stop stops the worker
 func (c *CallStatsServer) Stop() {
+	c.closed = true
```

Comment:
Do you need to modify this atomically?

Reply:
good point, may be, let me check, thanks

Reply:
I probably do, but I would hate to do just for that, I will leave it as is, hope you are ok with it.
if it crashes on close, be it.

Reply:
It should be relatively painless with a compare-and-swap operation on a uint, but up to you!

Reply:
the problem this is addressing should even be here, the root of the problem is dfc runners stop has no sequence, one example is after httprunner is stopped, there can still be calls pending, when they finish, they will call call stats, if the channel is closed too early, it causes an error, that's why I added this, in typical go code this should not happen.

Reply:
atomic is costly

Reply:
I just remove the closed flag, for now, let's see how often it happens,
my solution is half baked, probably as 'good' as no solution.